docker/docker-compose.yaml

```diff
@@ -19,4 +19,19 @@ services:
     container_name: igt
     privileged: true
     volumes:
-      - $log_dir:/tmp/results
\ No newline at end of file
+      - $log_dir:/tmp/results
+  # xpum:
```

Comment:
is this supposed to be commented?

Reply:
For now since XPUM is only required for Flex GPUs. We will add additional logic later to determine if we use IGT or XPUM